frontend/test/lit/test_for_loop.py

```diff
@@ -42,24 +42,19 @@ def loop_fn(i, phi):
         # CHECK:       [[i_cast:%.+]] = arith.index_cast [[i]]
         # CHECK:       [[phi1:%.+]] = stablehlo.add %arg3, %arg1
 
-        # CHECK:       [[q0:%.+]] = "quantum.extract"([[r0]], [[i_cast]])
+        # CHECK:       [[q0:%.+]] = quantum.extract [[r0]][[[i_cast]]]
         # CHECK:       [[phi_e:%.+]] = tensor.extract [[phi0]]
-        # CHECK:       [[q1:%.+]] = "quantum.custom"([[phi_e]], [[q0]]) {gate_name = "RY"
-        # CHECK:       [[r1:%.+]] = "quantum.insert"([[r0]], [[i_cast]], [[q1]])
+        # CHECK:       [[q1:%.+]] = quantum.custom "RY"([[phi_e]]) [[q0]]
+        # CHECK:       [[r1:%.+]] = quantum.insert [[r0]][[[i_cast]]], [[q1]]
         qml.RY(phi, wires=i)
 
         # CHECK:       scf.yield [[phi1]], [[r1]]
         return phi + inc
 
     loop_fn(0.0)
-    # CHECK:       "quantum.dealloc"([[qreg]])
+    # CHECK:       quantum.dealloc [[qreg]]
```

Comment:
Isn't this now inconsistent with the assembly format of alloc op, i.e., `quantum.alloc(3)`?

Reply:
On the contrary, this is consistent with the assembly format of `memref.alloc` and `dealloc`:
```mlir
%0 = memref.alloc() : memref<f64>
memref.dealloc %0 : memref<f64>
```
The assembly format also predates this PR and this PR does not modify it, it only modifies the test.